### ISSUE TITLE
Audio.load/play of inexisting file: wrong error

### ISSUE THREAD

**Issue by mougino** [RFO-BASIC/Basic#191]
In BASIC! v01.87, in Editor mode, if you `audio.load` an inexisting file, no error is thrown. Error is thrown only when you try to `audio.play` it:
> Invalid Player List Value
> audio.play musptr

Program:
```VB
audio.load musptr, "inexisting file.mp3"
audio.stop
audio.play musptr
```

**Comment by jMarcS** [comment]
`audio.load` does not throw an error if no file is found. It sets the list index to 0, which is an invalid list index. Other commands that try to take a player from the list get an error if the list index is 0.

It's just like `text.open` returning -1 if no file is found. There is no error unless you try to use the file index to read or write a file.

So there is no bug in the logic of the code. On the other hand, the error message from `audio.play` is not clear. It assumes you know what the "Player List" is, and what values are valid. That is, it assumes you are familiar with the Java implementation, not just the description in the manual.

Two changes:
- We must fix the manual so it tells the user what to expect and what the error means.
- We can change the error message text.

The error message can't say, "Audio file does not exist." The command doesn't know why your list index is out of range. It just knows 0 is invalid.

The manual refers to the "Audio File Table", or "AFT". That's the "Player List". So how about "Invalid Audio FIle Table index"? Good enough, or do we need something better?

**Comment by mougino** [comment]
Ok my bad, I found it was inconsistent with a `gr.bitmap.load` with an inexisting file (in this case it throws an error).
Shouldn't it be similar between the 2 functions ?

**Comment by mougino** [comment]
If it is the expected behavior, I am ok with the current error `Invalid Player List Value`, all the more when the incriminated line is highlighted, the user can see clearly what the problem is.
Just a note in the manual would be enough I think.

**Comment by mougino** [comment]
Marc, you can close this issue I was unaware it was designed like that. I don't want to add more on your plate, which is already quite full!

**Comment by jMarcS** [comment]
Definitely want it consistent with `gr.bitmap.load`: I did [this commit](https://github.com/RFO-BASIC/Basic/commit/3b9e8f737166c09dec56f07fd1228a806bf71533) in response to QBFan and magicjohn in [this thread](http://rfobasic.freeforums.org/error-message-bitmap-load-failed-at-t3417.html). That was only a few days ago.

**Comment by jMarcS** [comment]
Released in v01.88, 2015/08/16.
